package.json

```diff
@@ -25,7 +25,8 @@
     "publish-release": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish --conventional-graduate --exact || echo 'Error: New release can only be published on main branch' >&2",
     "publish-prerelease": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish prerelease --preid beta --dist-tag next --exact || echo 'Error: New release can only be published on main branch' >&2",
     "test": "jest",
-    "test-cli": "lerna run test-headless --stream",
+    "test-cli": "lerna run test-headless --stream --",
```

Comment:
The double dashes at the end of this allow trailing arguments to pass on to the next script. Now we can run this from the cli root directory with inline args